src/components/CMakeLists.txt

```diff
@@ -30,61 +30,62 @@
 
 set(COMPONENTS_DIR ${CMAKE_SOURCE_DIR}/src/components)
 
-# --- HMI_API interfaces
+# --- Interfaces
```

Comment:
@anosach-luxoft, please add `HMI and MOBILE API` for avoiding messing up with an C++ components interfaces in `components/include/` directory